docs/getting-started/new_project_guide.md

```diff
@@ -253,11 +253,8 @@ alphanumeric characters, underscore(_) or dash(-). Otherwise, they won't run on
 
 ### Temporarily disabling code instrumentation during builds
 
-Sometimes not every 3rd party library might be needed to be instrumented or
-tools are being compiled that just support the target built.
+In some cases, it's not necessary to instrument every 3rd party library or tool that supports the build target. Use the following code snippet to specify build commands that should not result in instrumented code:
```

Comment:
The second sentence here is a bit confusing. please reword the second sentence as 

"Use the following snippet to build tools or libraries without instrumentation."

Reply:
Done, thanks for the suggestion!